grades/templates/grading_systems.html

```diff
@@ -0,0 +1,17 @@
+<html>
+    <b>Default Grading System:</b>
```

Comment:
I feel you should add a button or link to go back to moderator dashboard. It is difficult to navigate back to moderator dashboard

Reply:
I understand! But I have not kept Yaksh moderator dashboard link as its a separate grades app.
I m working on a way that it picks the base template of Yaksh by default.
It is include in issue #437 related to good UI

Reply:
For now you can put a link with href having URL_ROOT/exam so that it redirects to dashboard later on you can change UI and make it better. Currently I have to hit back from browser back button.

Reply:
I am working in this with better implementation, will send with next PR. Not a major problem with the moderator.